.github/workflows/base.yaml

```diff
@@ -0,0 +1,19 @@
+name: "Build Base"
+run-name: Build Base of ${{ github.ref_name }} by @${{ github.actor }}
+on:
+  workflow_dispatch:
+    inputs:
+      debug_enabled:
+        type: boolean
+        description: 'Run the build with tmate debugging enabled (https://github.com/marketplace/actions/debugging-with-tmate)'
+        required: false
+        default: false
+
+jobs:
+  build:
+    uses: scientist-softserv/actions/.github/workflows/build.yaml@v0.0.12
+    secrets: inherit
+    with:
+      platforms: "linux/amd64" # "linux/amd64,linux/arm64"
+      target: hyku-base
+      image_name: samvera/hyku/base
```

Comment:
we will need to update these to pals specific

Reply:
Does Pals even use this file?

Reply:
matched the actions version